robot/description/urdf/constants.xacro

```diff
@@ -18,14 +21,67 @@
     <xacro:property name="caster_link_to_wheel_dist_z" value="-0.04361749"/>
 
     <!-- Wheels -->
+    <!-- TODO: update parameters once they become known -->
```

Comment:
these values shouldn't be unknown, which ones are unknown?  (if none, then remove the comment)

@kajananchinniah if there are any values that you are setting as default bc you don't know what they should be (and we reasonably can't know atm), leave a not saying something like `using default values` that concerns the section of interest

Reply:
The unknown parameters are:
`wheel_acceleration`
`wheel_torque`